<launch>
  <!-- a robot description MUST be on the param server for this node to work -->
  <node pkg="sr_arm_kinematics" type="sr_arm_kinematics6D" name="shadow_right_arm_kinematics" output="screen">
    <param name="root_name" type="string" value="shadowarm_base"/>
    <param name="tip_name" type="string" value="palm"/>
    <param name="maxIterations" type="int" value="1000"/>
    <param name="epsilon" type="double" value="0.001"/>
  </node>
</launch>


